ui/src/app/views/notebook-list/component.ts

```diff
@@ -212,13 +212,13 @@ export class NotebookListComponent implements OnInit, OnDestroy {
 
   receiveDelete($event: FileDetail): void {
     this.deleteNotebook($event);
-    this.deleteModal.close();
   }
 
   public deleteNotebook(notebook: FileDetail): void {
```

Comment:
Why is this a public method? Looks like it may be a historical artifact as it doesn't seem to be referenced anywhere else. I would just inline this method into the `receiveDelete` above; as it stands it's a bit messy that `deleteNotebook` would have the logic for closing the dialog (alternatively, if this method is still needed for some reason, return the observable here and add the deleteModal callback within `receiveDelete`).

Reply:
Yeah, I think its a historical artifact. Will change.